integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -541,7 +544,7 @@ public int readdir(String path, Pointer buff, FuseFillDir filter,
         filter.apply(buff, file.getName(), null, 0);
       }
     } catch (FileDoesNotExistException | InvalidPathException e) {
-      LOG.debug("Failed to read directory {}, path is invalid or does not exist", path, e);
+      LOG.debug("Failed to read directory {}, path does not exist or is invalid", path, e);
```

Comment:
Do we want to provide the full exception in the log? Currently we do it from places and not for others. I would prefer to not log the entire exception (ie. `LOG.debug("Failed to read directory {}, path does not exist or is invalid", path);`

Reply:
Removed the full exception in debug level logging, thanks